core/controllers/profile.py

```diff
@@ -89,8 +89,15 @@ def get(self, username):
             edited_exp_summary_dicts = (
                 summary_services.get_displayable_exp_summary_dicts_matching_ids(
                     user_contributions.edited_exploration_ids))
+            ### function to determine whether current user is same as user on page ###
```

Comment:
Yes please, typically the rule of thumb is that unless it is super complicated and obscure, you don't need to comment. (Also teaches you to write readable code!)